### DISCUSSION TITLE
مشکل inactive بودن hiddify-haproxy

### DISCUSSION THREAD

**Discussion by rasoulfattahi** [hiddify/Hiddify-Manager#3837] [Q&A سوالات رایج]
hiddify-cli                   inactive
  hiddify-haproxy               inactive
    
   سلام برای اولین بار پنل رو نصب میکنم و دو گزینه بالا غیر فعاله با اپدیت و اینجور کارا هم درست نشد اگر میتونید ممنون میشم کمک کنید

**Comment by D-7J** [comment]
> hiddify-cli inactive hiddify-haproxy inactive
> 
> سلام برای اولین بار پنل رو نصب میکنم و دو گزینه بالا غیر فعاله با اپدیت و اینجور کارا هم درست نشد اگر میتونید ممنون میشم کمک کنید

`apt install haproxy -y`
`cd /opt/hiddify-manager/other/hiddify-cli`
`chmod +x install.sh`
`./install.sh`
بعدش دوباره چک کن ببین درست شده یا نه یه بار هم ریبوت کن ببین همه چی اوکیه یا نه !
